assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -3,6 +3,16 @@
   width: auto;
   max-width: 100%;
 
+  figure:not(.alignleft):not(.alignright) {
```

Comment:
Do you think it's possible to make this rule less specific? Because it would be very difficult to be overridden by child themes. Maybe add these rules for all `figure` and then override for `.alignleft` and `.alignright`?

Reply:
@comzeradd Updated!